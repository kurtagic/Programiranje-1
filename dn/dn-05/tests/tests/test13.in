50
2 34
1 39
0 52
1 51
0 69
2 62
0 48
1 76
0 46
2 50
2 36
1 62
0 35
1 72
0 40
2 40
1 70
2 79
2 55
2 31
0 54
0 74
1 51
2 44
2 49
1 37
1 38
2 73
1 66
1 80
0 63
2 67
1 67
1 42
2 62
0 62
0 63
1 71
0 44
0 60
0 64
0 58
1 63
2 63
2 69
2 76
1 47
2 41
1 58
0 31
