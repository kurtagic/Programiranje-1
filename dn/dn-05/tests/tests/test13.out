31: 3
32: 3
33: 3
34: 3
35: 3
36: 3
37: 7
38: 7
39: 4
40: 3
41: 7
42: 7
43: 1
44: 7
45: 7
46: 7
47: 11
48: 6
49: 9
50: 9
51: 11
52: 1
53: 1
54: 4
55: 4
56: 4
57: 4
58: 16
59: 12
60: 12
61: 12
62: 11
63: 17
64: 17
65: 11
66: 11
67: 11
68: 2
69: 7
70: 7
71: 5
72: 5
73: 7
74: 7
75: 5
76: 7
77: 7
79: 2
80: 6
